lib/types/endpoints.js

```diff
@@ -74,6 +74,7 @@ const socketPreferredEndpoints = Object.freeze({
   REQUEST_ACCESS: 'request_access',
   CREATE_MULTIMEDIA_MESSAGE: 'create_multimedia_message',
   DELETE_UPLOAD: 'delete_upload',
+  SEND_FRIEND_REQUEST: 'send_friend_request',
```

Comment:
Let's get rid of these too, since we're not planning on actually creating this endpoint